spec/lib/hubstats/github_api_spec.rb

```diff
@@ -61,7 +61,7 @@ module Hubstats
 
         it 'fails to initialize at all' do
           Hubstats::GithubAPI.configure()
-          expect(lambda { Hubstats::GithubAPI.client}).to raise_error Octokit::Unauthorized
+          expect{Hubstats::GithubAPI.client}.to raise_error Octokit::Unauthorized
```

Comment:
When I put the allow, I would ironically get the unauthorized error. So I suppose I don't know 100% that this solved the error. It's very possible it just stopped giving me the TooManyRequests error.